manifests/r/RStudio/quarto/1.0.38/RStudio.quarto.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.0.38
+PackageLocale: en-US
+Publisher: Rstudio, PBC
+PublisherUrl: https://www.rstudio.com/
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Quarto
+PackageUrl: https://quarto.org/
+License: GPL v2
+LicenseUrl: https://quarto.org/license.html
+Copyright: Copyright (C) RStudio, PBC
+CopyrightUrl: https://quarto.org/trademark.html
+ShortDescription: Quarto® is an open-source scientific and technical publishing system built on Pandoc
+Description: "Main features of Quarto: \n  Create dynamic content with Python, R, Julia, and Observable.\n  Author documents as plain text markdown or Jupyter notebooks.\n  Publish high-quality articles, reports, presentations, websites, blogs, and books in HTML, PDF, MS Word, ePub, and more.\n  Author with scientific markdown, including equations, citations, crossrefs, figure panels, callouts, advanced layout, and more.\n"
```

```suggestion
Description: |-
  Main features of Quarto:
     Create dynamic content with Python, R, Julia, and Observable.
     Author documents as plain text markdown or Jupyter notebooks.
     Publish high-quality articles, reports, presentations, websites, blogs, and books in HTML, PDF, MS Word, ePub, and more.
     Author with scientific markdown, including equations, citations, crossrefs, figure panels, callouts, advanced layout, and more.
```
You probably don't want `\n` to be rendered literally